### PR TITLE
minor fixes in the publishing tracing

### PR DESCRIPTION
**Pull request by Kunal-Jha**
# minor fixes in the publishing tracing
Zalando ticket: ARUHA-2467

## Description
Minor fixes

https://jira.zalando.net/browse/ARUHA-2547

### CODE REVIEW

File: src/main/java/org/zalando/nakadi/filters/LoggingFilter.java [hunk 1]
```diff
@@ -243,4 +244,10 @@ private boolean isPublishingRequest(final RequestLogInfo requestLogInfo) {
                 requestLogInfo.path.startsWith("/event-types/") &&
                 (requestLogInfo.path.endsWith("/events") || requestLogInfo.path.endsWith("/events/"));
     }
+
```
Comment (rcillo) [inline src/main/java/org/zalando/nakadi/filters/LoggingFilter.java]: I guess you are going to use this in the next PR right?

**Comment by rcillo** [comment]
👍

File: src/main/java/org/zalando/nakadi/filters/LoggingFilter.java (hunk 1 above)
Comment (Kunal-Jha) [reply src/main/java/org/zalando/nakadi/filters/LoggingFilter.java]: Yes

**Comment by Kunal-Jha** [comment]
👍

**Pull request merged**